_posts/2022-11-02-relicensing.md

```diff
@@ -6,6 +6,33 @@ title: Relicensing MDAnalysis
 <img src="https://www.gnu.org/graphics/lgplv3-with-text-154x68.png"
 title="LGPLv3" alt="LGPLv3 logo" style="float: right"/>
 
+> :warning: **Disclaimer**: The MDAnalysis core team members are not
+> lawyers, as such the information provided here __does not, and is not
+> intended to, constitute legal advice__. This blog post also does not
+> represent MDAnalysis' full legal position on software licensing, it
+> purely aims to inform MDAnalysis developers and users on why
+> we believe the library should be relicensed.
+>
+> Further information on open-source software licensing can be found
+> from sources such as the [Open Source Initiative][OSI] and the
+> [Software Sustainability Insitute][SSI licensing].
+>
+> Should you have any concerns about licensing, we always strongly
+> recommend getting legal advice before making any decisions on how
+> licensing changes may affect you.
+
+
+## Short summary
```

Comment:
Because everyone is busy, especially for complicated subjects such as licensing, I'm proposing we have short TLDR-style 2 sentence summaries like this. It help folks get the quick gist of what's happening and whether they need to be invested in it.

Reply:
I like TLDR. I would just start the post with it and not bother calling it "Short summary". By virtue of being the first thing that people read it fulfills its function.